spec/features/pipelines/pipelines_test.go

```diff
@@ -7,8 +7,22 @@ import (
 )
 
 func TestRunTutorial(t *testing.T) {
-
 	Convey("Given that Operator is installed", t, func() {
 		Convey("I should be able to run Pipelines Tutorial as a non admin", nil)
 	})
 }
+
+func TestPipelineRunForClusterTasks(t *testing.T) {
+	Convey("Given that Operator is installed", t, func() {
+		Convey("I should be able to run pipeline successfuly", func() {
```

Comment:
this needs to change as well to follow ... 

Given
When
Then 

Instead of 
Given 
then
When